casper-feed/src/main/kotlin/hs/kr/entrydsm/feed/global/utils/user/UserUtils.kt

```
@@ -0,0 +1,28 @@
+class UserUtils {
+
+    /**
+     * 현재 인증된 사용자의 고유 식별자(UUID)를 조회합니다.
+     *
+     * @return 현재 인증된 사용자의 UUID
+     * @throws hs.kr.entrydsm.feed.global.exception.InvalidTokenException 인증 토큰이 유효하지 않은 경우
+     */
+    fun getCurrentUserId(): UUID {
+        try {
+            return UUID.fromString(SecurityContextHolder.getContext().authentication.name)
+        } catch (e: IllegalArgumentException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)